<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialEditorClass</class>
 <widget class="QMainWindow" name="MaterialEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>218</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>218</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MaterialEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Material Properties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Color</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Blue :</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Red : </string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Green :</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="cred">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="cgreen">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="cblue">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QGraphicsView" name="prevc">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Specular</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Green :</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Blue :</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Red :</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Exponent :</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="sred">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="sgreen">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="sblue">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="sexp">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QGraphicsView" name="prevs">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>218</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Material"/>
    <addaction name="actionSave_Material"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Material">
   <property name="text">
    <string>New Material</string>
   </property>
  </action>
  <action name="actionSave_Material">
   <property name="text">
    <string>Save Material</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MaterialEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cred</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cgreen</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cblue</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sred</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sgreen</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sblue</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sexp</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>MarkChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cred</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>UpdateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cgreen</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>UpdateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cblue</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>UpdateColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sred</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>UpdateSpecColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sgreen</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>UpdateSpecColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sblue</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MaterialEditorClass</receiver>
   <slot>UpdateSpecColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MarkChanged()</slot>
  <slot>UpdateColor()</slot>
  <slot>UpdateSpecColor()</slot>
 </slots>
</ui>
